reference-sheets/ControlTheory.md

```diff
@@ -0,0 +1,32 @@
+# Introduction
+
+Control theory is a widely-used concept in engineering that helps influence the behavior of a system to achieve a desired outcome. By applying methods like feedback and feedforward, we can correct errors and make systems more accurate and reliable. This guide covers the basics of these methods but primarily provides resources with more in-depth explanations of each topic. Many of the projects you will work on will implement these concepts, so it's important to understand how they function.
+
+To get started, read [this](https://docs.wpilib.org/en/stable/docs/software/advanced-controls/introduction/control-system-basics.html) document on control systems. It shows how control systems are used in everyday life and why they’re important. It also breaks down basic concepts and uses diagrams to explain how most systems work.
+
+### Open vs. closed loop control
+
+**Open-loop control**, commonly known as `feedforward`, functions by sending control signals based on predetermined estimates, not by adjusting for the system's current state. This means it operates without feedback, relying only on initial assumptions about how the mechanism should behave. As seen below, feedforward is usually depicted as a straight path regardless of the "loop" in its name.
```

Comment:
There should be a discussion on _both_ types of FRC feedforwards (plant inversion & accounting for additional forces). Shouldn't be too in-depth (beginners don't need to know plant inversion) but they should know it exists. 

You can link to Controls Engineering in FRC for further exploration. With that said, we should also have a resources doc